client/src/pages/Contest/PreferedDesigns/DesignItem/DesignItem.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { FunctionComponent } from 'react';
+import Card from '@material-ui/core/Card';
+import Box from '@material-ui/core/Box';
+import Grid from '@material-ui/core/Grid';
+import useStyles from './useStyles';
+import CheckCircleOutlineIcon from '@material-ui/icons/CheckCircleOutline';
+import img from '../../../../Images/68f55f7799df6c8078a874cfe0a61a5e6e9e1687.png';
+import { boolean } from 'yup';
+import { string } from 'yup/lib/locale';
+
+interface Props {
+  isSelected: boolean;
+  imgSrc: string;
+}
+
+const DesignItem: FunctionComponent<Props> = ({ isSelected, imgSrc }: Props): JSX.Element => {
```

Comment:
Nice thinking componentizing this. :)

For future use, you may want to look into Image Lists in MUI 
Ref: https://next.material-ui.com/components/image-list/

Reply:
I can not believe I missed this component...  Doh.

I'll look into it.

Reply:
Looks like ImageList isn't in the version of MUI we are using.  It was GridList previously and not quite the same.
Any thoughts on upgrading?

Reply:
Sorry, I must've missed this question. Please feel free to upgrade to a later version of MUI 👍🏽